src/lib/services/user-service.ts

```diff
@@ -136,28 +136,32 @@ class UserService {
     ): Promise<void> {
         let username: string;
         let password: string;
+        let email: string;
 
         if (
             initialAdminUserConfig.createAdminUser !== false &&
             initialAdminUserConfig.initialAdminUser
         ) {
             username = initialAdminUserConfig.initialAdminUser.username;
             password = initialAdminUserConfig.initialAdminUser.password;
+            email = initialAdminUserConfig.initialAdminUser.email;
         } else {
             username = 'admin';
             password = 'unleash4all';
+            email = 'admin@unleash.com';
         }
 
         const userCount = await this.store.count();
 
-        if (userCount === 0 && username && password) {
+        if (userCount === 0 && username && password && email) {
```

Comment:
I'm not sure we actually need to check for the existence of the email here? The only time this'll be falsy is if you provide an empty string as an env variable, right, but still ask unleash to create the initial admin user? 

I don't think we should **not** create the admin user if you provide an empty email 🤔

Reply:
+1 good call imo

Reply:
+1